app/tasks/schedule.php

```diff
@@ -3,12 +3,14 @@
 global $cli;
 global $register;
 
+use Appwrite\Event\Event;
 use Cron\CronExpression;
 use Utopia\App;
 use Utopia\CLI\Console;
 use Utopia\Database\DateTime;
 use Utopia\Database\Query;
 use Swoole\Timer;
+use Utopia\Queue\Client as worker;
```

Comment:
Lets use Title case when importing classes

Reply:
Not renaming anymore.